frontend/public/components/image-stream-tag.tsx

```diff
@@ -110,17 +129,19 @@ export const SupportedPlatformsTable = (props) => {
 
   return (
     <>
-      {props.heading && <SectionHeading text={props.heading} />}
-      <Table
-        {...tableProps}
-        aria-label={t('public~Supported Platforms')}
-        loaded={true}
-        label={props.heading}
-        data={submanifests}
-        Header={SupportedPlatformsTableHeader}
-        Rows={SupportedPlatformsTableRows}
-        virtualize={false}
-      />
+      {heading && <SectionHeading text={heading} />}
+      <React.Suspense fallback={<LoadingBox />}>
+        <ResourceDataView
+          label={t('public~Supported Platforms')}
```

Comment:
I don't see a `Supported Platforms` table on ImageStreamTag details page either on master branch or PR cluster, I can only see `Image labels` and `Environment variables` section
is there any per-requisite to display it?  
<img width="1222" height="743" alt="Screenshot 2025-10-21 at 11 35 13 AM" src="https://github.com/user-attachments/assets/1a0120c2-0d33-48c8-b4c4-6bc00ecdec2b" />